src/components/Reports/ProjectReport/ProjectReport.jsx

```diff
@@ -29,9 +34,28 @@ export const ProjectReport = ({ match }) => {
     }
   }, []);
 
+  useEffect(() => {
```

Comment:
In here we are fetching all tasks 3 times. We can use the fetching boolean property of wbs to avoid sending unnecessary duplicate requests to the backend:

`if (wbs.fetching === false) {
      wbs.WBSItems.forEach(wbs => {
        dispatch(fetchAllTasks(wbs._id));
      });
    }`

Reply:
By the way, I am using Morgan npm package to log all API calls being made to the backend.

Reply:
Great catch Diego! These are some great optimizations and I will write this on the docs and do a follow-up PR this week. Thank you.